test-data/unit/check-errorcodes.test

```diff
@@ -127,7 +127,7 @@ x
 y # type: ignore  # ignore the lack of error code since we're ignore the whole file
```

Comment:
```suggestion
y # type: ignore  # ignore the lack of error code since we ignore the whole file
```
(not related but might as well fix the grammar here)